pisek/defaults-config

```diff
@@ -65,3 +65,17 @@ sec_solve_process_limit=1
 
 input_max_size=50
 output_max_size=10
+
+[cms]
+title=
+submission_format=
```

Comment:
Isn't better to use autoexpand values (``@``) for this? As empty string is also technically value (although invalid one).